packages/usa-footer/src/styles/_usa-footer.scss

```diff
@@ -22,103 +22,102 @@ $-chevron-expand-more: map-merge(
   )
 );
 
-// General footer styles
-.usa-footer {
-  @include border-box-sizing;
-  @include typeset($theme-footer-font-family);
-  overflow: hidden;
+// Grid styles ported over to remove dependency
+.grid-row {
+  @include grid-row;
+}
 
-  // Grid styles ported over to remove dependency
-  .grid-container {
-    @include grid-container($theme-footer-max-width);
-  }
+// Gaps
+.grid-gap {
+  @include grid-gap;
+}
 
-  .grid-row {
-    @include grid-row;
+.grid-gap-1 {
```

Comment:
## Optional
This is a small note, but I am wondering if we should force the output to match the other grid gap CSS to be attached to `.grid-row`: `.grid-row.grid-gap-[X]`. Then if anyone has a compiler that removes redundant code, it will remove these lines.

Reply:
Ok so I nested the classes using:

```scss
.grid-row {
   &.grid-gap-[x]
```

This seemed to properly duplicate the `.grid-row.grid-gap-[x]` classes when comparing the output css!

No observed visual regressions or style conflicts as a result 👍